eng/common/instructions/azsdk-tools/local-sdk-workflow.instructions.md

```diff
@@ -117,3 +123,29 @@ Help the user generate and build SDKs locally from TypeSpec API specifications u
 **Actions**:
 
 - Run `azsdk_package_build_code` MCP tool to compile the SDK in the identified project directory.
+
+---
+
+### Step 3: Run package validation
+
+**Actions**:
+
+- Run `azsdk_package_run_check` MCP tool to validate the generated SDK package in the identified project directory.
+
+---
+
+### Step 4: Run package tests
+
+**Actions**:
+
+- Run `azsdk_package_run_tests` MCP tool to run tests on the generated SDK package in the identified project directory.
+
+---
+
+### Step 5: Update change log, metadata and version
```

Comment:
Do we want to split these tools in separate steps? Not sure which approach is preferred for the agent, but the point is the execution sequence of these tools needs to remain fixed.

Reply:
I think it's better to keep it under one main step with substeps in correct order.